src/app/lib/config/services/hausa.js

```diff
@@ -123,29 +123,80 @@ export const service = {
           rejectUrl: 'https://www.bbc.co.uk/usingthebbc/your-data-matters',
         },
         cookie: {
-          title: "Bayyana mana idan ka amince da ka'idojin",
-          description: {
-            uk: {
-              first: 'Muna amfani da ',
-              linkText: "ka'idoji",
-              last:
-                " domin samar maku da abubuwa masu kayatarwa a Intanet. Muna rokonku da ku sanar da mu idan kun gamsu da duka wadannan ka'idoji.",
-              linkUrl:
-                'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
+          amp: {
+            accept: 'Ku amince da tattara bayananku sannan ku ci gaba',
+            reject: 'Ƙin amincewa da tattara bayananku sannan ku ci gaba',
+            initial: {
+              title:
+                'Masu ziyarar shafin AMP ku sanar da mu idan kun amince mu tattara bayananku',
+              description: {
+                first:
+                  'Mu da sauran abokan hulda muna amfani da fasaha kamar adireshin waje, sannan mu tattara wasu bayanai game da ku duka domin mu samar maku da abubuwa masu kayatarwa a Intanet',
+                linkText: '',
+                last: '',
```

Comment:
Do we not have translations for these?

Reply:
Hey @andrewscfc, do you know about this by any chance?

Reply:
I _think_ for some mysterious reason hausa doesn't have a link? This was queried but no answer came I think